lib/matplotlib/sankey.py

```diff
@@ -739,7 +739,13 @@ def _get_angle(a, r):
             if label is None or angle is None:
                 label = ''
             elif self.unit is not None:
-                quantity = self.format % abs(number) + self.unit
+
+                if isinstance(self.format, str):
+                    quantity = self.format % abs(number) + self.unit
+                elif callable(self.format):
+                    quantity = self.format(100. * abs(number))
```

Comment:
I'm skeptical you want to multiply by 100 here?

Reply:
Sorry. First PR and I derped. 

I think what happened was that this I was confused or missed a step between committing to local branch and pushing master branch on my fork. There seems to be two steps.

Thanks for catching the error.